modules/oadp-preparing-aws-credentials.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * rosa_backing_up_and_restoring_applications/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-preparing-aws-credentials_{context}"]
+= Preparing AWS credentials
+
+An AWS account must be ready to accept an OADP installation.
+
+.Procedure
+. Create the following environment variables by running the following commands:
++
+[NOTE]
+====
+Change the cluster name to match your ROSA cluster, and ensure you are logged into the cluster as an administrator. Ensure that all fields are outputted correctly before continuing.
+====
++
+[source,terminal]
+----
+$ export CLUSTER_NAME=my-cluster <1>
+export ROSA_CLUSTER_ID-$(rosa describe cluster -c ${CLUSTER_NAME} --output json | jq -r .id)
+export REGION=$(rosa describe cluster -c ${CLUSTER_NAME} --output json | jq -r .region.id)
+export OIDC_ENDPOINT=$(oc get authentication.config.openshift.io cluster -o jsonpath='{.spec.serviceAccountIssuer}' | sed 's|^https://||')
+export AWS_ACCOUNT_ID='aws sts get-caller-identity --query Account --output text'
+export CLUSTER_VERSION='rosa describe cluster -c ${CLUSTER_NAME} -o json | jq -r .version.raw_id | but -f -2 -d '.' '
+export ROLE_NAME="${CLUSTER_NAME}-openshift-oadp-aws-cloud-credentials"
+export SCRATCH="/tmp/${CLUSTER_NAME}/oadp"
+mkdir -p ${SCRATCH}
+echo "Cluster ID: ${ROSA_CLUSTER_ID}, Region: ${REGION}, OIDC Endpoint: 
+${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
+----
++
+<1> Replace `my-cluster` with your ROSA cluster name.
+
+. On the AWS account, create an IAM policy to allow access to S3 by running the following command:
++
+[source,terminal]
+----
```

Comment:
In the module above, you used subtasks for a step that had multiple tasks. Would you want to do that here? Eg:

```suggestion
. On the AWS account, create an IAM policy to allow access to S3. 
.. Run the following command to do this policy step:
+
[source,terminal] 
----
$ POLICY_ARN=$(aws iam list-policies --query "Policies[?PolicyName=='RosaOadpVer1'].{ARN:Arn}" -- output text)
----
.. Next step
.. Next step 
```

Reply:
@weshayutin good suggestion by @stevsmit, because we separated the code block into 3 parts we should add in subtasks for each separated code. Can you advise the term we can use to describe each if the 3 subtasks for create an IAM policy.
a. 
b.
c.
![image](https://github.com/openshift/openshift-docs/assets/97602793/24957a28-44cb-4a33-b1dd-44d3ec436160)
![image](https://github.com/openshift/openshift-docs/assets/97602793/9a5c94b5-52c8-4133-b1dd-9a466373952a)

Reply:
@weshayutin and @kaovilai if you had time to comment please on the info describing the separated commands please and if we need the `$` in front of sub tasks (a. b. c.), thanks

Reply:
If you're referring to $ in the first character of commands in a, and b (c doesn't have $ as first char) then no, don't need it.

If we remove it, we should be consistent in the rest of the page.

Reply:
to your standards.. it's actually should be 4
the first three you listed here.. and one additional

```
$ echo ${POLICY_ARN}
```

Long live the "$"

Reply:
applied with @weshayutin who kindly assisted me